### ISSUE TITLE
Delete or transfer old hls GH Classic Project

### ISSUE THREAD

**Issue by fzadrazil** [hslayers/hslayers-ng#4693]
There is an old GitHub projects classic https://github.com/orgs/hslayers/projects/1
Is it still relevant or should be replaced by some other repository of existing apps?

**Comment by fzadrazil** [comment]
Yep, I agree and that was basically the reason why I created that ticket :-) I had no idea about that GH project before. I will try to come up with some ideas where to store the list of apps. If you have any hints, feel free to propose.

**Comment by jmacura** [comment]
> Yep, I agree and that was basically the reason why I created that ticket :-) I had no idea about that GH project before. I will try to come up with some ideas where to store the list of apps. If you have any hints, feel free to propose.

Neither I had. I guess a "stupid" Google sheet or similar thing in our Nextcloud would do the job 🤔 Alternatively, we can create a new [wiki-page](https://github.com/hslayers/hslayers-ng/wiki) for it, if it's OK to be public.

**Comment by fzadrazil** [comment]
I guess Wiki page is good enough. https://github.com/hslayers/hslayers-ng/wiki/Sample-HSLayers%E2%80%90ng-applications

Once we migrate the official web, we can review the list of sample apps and this one and decide which applications do we need to maintain.
